doc/project_governance/code_of_conduct.md

```diff
@@ -0,0 +1,161 @@
+In person reports will still be kept confidential exactly as above, but also feel free to (anonymously if needed) email conduct@lowRISC.org.
+
+## Filing a report
+
+Reports can be as formal or informal as needed for the situation at hand.
+If possible, please include as much information as you can. If you feel comfortable, please consider including:
+
+* Your contact info (so we can get in touch with you if we need to follow up).
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+## What happens after you file a report?
+
+You will receive an email from the Code of Conduct team acknowledging receipt within 24 hours (and we will aim to respond much quicker than that).
+The Code of Conduct team will immediately meet to review the incident and try to determine:
+
+* What happened and who was involved.
+* Whether this event constitutes a code of conduct violation.
+* Whether this is an ongoing situation, or if there is a threat to anyone’s physical safety.
+* Whether the incident requires escalation to the OpenTitan Governing Board
+
+If this is determined to be an ongoing incident or a threat to physical safety, the working groups’ immediate priority will be to protect everyone involved.
+This means we may delay an “official” response until we believe that the situation has ended and that everyone is physically safe.
+The Code of Conduct team will try to contact other parties involved or witnessing the event to gain clarity on what happened and understand different perspectives.
+Once the Code of Conduct team has a complete account of the events they will make a decision as to how to respond.
+The Code of Conduct team may make any of the responses below, notifying the OpenTitan Governing Board of the action taken:
+
+* Nothing, if we determine no violation occurred or it has already been appropriately resolved.
+* Providing either moderation or mediation to ongoing interactions (where appropriate, safe, and desired by both parties).
+* A private reprimand from the working group to the individuals involved.
+
+In more serious cases the Code of Conduct team will escalate a proposed response to the OpenTitan Governing Board, with the final decision and action resting with the Governing Board.
+
+* An imposed vacation (i.e. asking someone to “take a week off” from a mailing list or IRC).
+* A public reprimand.
+* A permanent or temporary ban from some or all OpenTitanspaces (mailing lists, chat, events, etc.)
```

Comment:
I think you want a space before "spaces" (because I don't think we've got that neologism yet :-D)